app/blast/blast.py

```diff
@@ -0,0 +1,194 @@
+from contextlib import asynccontextmanager
+from urllib import response
+from fastapi import FastAPI, Response
+from fastapi.encoders import jsonable_encoder
+from fastapi.exceptions import RequestValidationError
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel
+from aiohttp import ClientSession, client_exceptions
+from enum import Enum
+from uuid import UUID
+import asyncio
+import secrets
+import json
+import re
+import os
+
+from core.config import BLAST_CONFIG
+
+
+# @asynccontextmanager
+# async def lifespan(app: FastAPI):
+#     # Startup: setup data cache and requests session
+#     app.client_session = ClientSession()
+#     with open("/data/blast_config.json") as f:
+#         app.blast_config = json.load(f)
+#     yield
+#     # Shutdown: close requests session
+#     await app.client_session.close()
```

Comment:
I would suggest deleting this commented out block. We can ask @veidenberg if he envisions any problems with migrating the client session from this function to the `run_blast` function.